tools/stress-cluster/services/Stress.Watcher/src/PodEventHandler.cs

```diff
@@ -164,10 +164,15 @@ public async Task StartChaosResources(V1Pod pod)
                         .Where(cr => ShouldStartChaos(cr, pod))
                         .Select(async cr =>
                         {
+                            string plural = "";
+                            foreach (string pluralName in Enum.GetNames(typeof(GenericChaosClient.ChaosResourcePlurals))) {
+                                if (pluralName.Contains(cr.Kind.ToLower())) {
+                                    plural = pluralName;
+                                }
+                            }
                             await Client.PatchNamespacedCustomObjectWithHttpMessagesAsync(
                                     PodChaosResumePatchBody, ChaosClient.Group, ChaosClient.Version,
-                                    pod.Namespace(), cr.Kind.ToLower(), cr.Metadata.Name);
-
+                                    pod.Namespace(), plural, cr.Metadata.Name);
```

Comment:
Is this right? We update an individual object here so will the plurals API route still work?

Reply:
Yup, positive. I switched to plural because "schedule" doesn't work. it needs "schedules". Before chaos-mesh 2.1.5, it was fine because plural and singular for XXXChaos are the same.